### PR TITLE
FileResponse: added some checks

### PR DESCRIPTION
**Pull request by tacoberu**
- bug fix? yes
- BC break? no

If the file has strict privileges, it will crash the entire application.

### CODE REVIEW

File: src/Application/Responses/FileResponse.php [hunk 1]
```diff
@@ -40,6 +40,9 @@ public function __construct(string $file, string $name = null, string $contentTy
 		if (!is_file($file)) {
 			throw new Nette\Application\BadRequestException("File '$file' doesn't exist.");
 		}
+		if (!is_readable($file)) {
+			throw new Nette\Application\BadRequestException("File '$file' doesn't readable.");
```
Comment (dg) [inline src/Application/Responses/FileResponse.php]: is not readable sounds better

File: src/Application/Responses/FileResponse.php [hunk 2]
```diff
@@ -88,6 +91,9 @@ public function send(Nette\Http\IRequest $httpRequest, Nette\Http\IResponse $htt
 
 		$filesize = $length = filesize($this->file);
 		$handle = fopen($this->file, 'r');
+		if (!$handle) {
+			throw new Nette\Application\BadRequestException("Cannot make handle of '{$this->file}'.");
+		}
```
Comment (dg) [inline src/Application/Responses/FileResponse.php]: Cannot open file?

File: src/Application/Responses/FileResponse.php (hunk 1 above)
Comment (JanTvrdik) [reply src/Application/Responses/FileResponse.php]: I'm not sure how useful the `is_readable` check is when you check `fopen` result later anyway.

Comment (tacoberu) [reply src/Application/Responses/FileResponse.php]: @JanTvrdik The sooner the better :-) But I don't insist on it.

**Pull request merged**